code/modules/mob/living/carbon/human/life.dm

```diff
@@ -17,8 +17,7 @@
 	life_tick++
 
 	voice = GetVoice()
-	if(stat == DEAD && iszombie(src) && regenZ)
-		handle_organs()
+	if(stat == DEAD)
```

Comment:
This is conflict with Spooky's life PR. Keep an eye on it.

Reply:
Also it will definitely conflict with all of my attackby PRs

Reply:
And done.